flake.nix

```diff
@@ -141,15 +142,20 @@
 
           src = self;
 
-          buildInputs =
-            [
+          nativeBuildInputs =
+            with final.buildPackages; [
```

Comment:
```suggestion
          nativeBuildInputs = with final.buildPackages; [
```

Reply:
I don't like the existing style either, but if I did that I would be better off moving way more things around?

Reply:
hmmm but that would also be a much bigger change and I didn't see the other lines in the diff 😉